crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -744,6 +819,7 @@ impl TryFrom<&HistoricRoomKey> for InboundGroupSession {
             // TODO: How do we remember that this is a historic room key and events decrypted using
             // this room key should always show some form of warning.
             sender_data: SenderData::default(),
+            forwarder_data: None,
```

Comment:
Is this impl still used? I thought the whole point of `try_into_inbound_group_session` was to get rid of this `fowarder_data: None` ?